mLRS/Common/hal/esp/rx-hal-dev-900-esp32.h

```diff
@@ -0,0 +1,111 @@
+//*******************************************************
+// Copyright (c) MLRS project
+// GPL3
+// https://www.gnu.org/licenses/gpl-3.0.de.html
+//*******************************************************
+// hal
+//********************************************************
+
+//-------------------------------------------------------
+// ESP8285, ELRS GENERIC 900 RX
+//-------------------------------------------------------
+
+#define DEVICE_HAS_SINGLE_LED
+//#define DEVICE_HAS_NO_DEBUG
+//#define DEVICE_HAS_SERIAL_OR_DEBUG
+
+
+//-- Timers, Timing, EEPROM, and such stuff
+
+#define SYSTICK_TIMESTEP          1000
```

Comment:
fyi: these two are btw not needed anymore in the hal files

Reply:
Removed